tests/reporting.tests.ts

```diff
@@ -122,46 +121,30 @@ describe('Reporting tests', () => {
     })
     describe('analytics', async () => {
         it('can get analytics history for a site', async () => {
-            scope.get(`/api/analytics/site/${site_name}`, (query) => {
-                expect(query).to.eql({
-                    from: from,
-                    to: to,
-                    dimension: dimension,
-                    result: result,
-                    dateGranularity: dateGranularity
-                })
-                return query
-            }).reply(200, analyticsResponse)
+            scope.get(`/api/analytics/site/${site_name}?from=2021-01-01&to=2021-12-01&dimension=system&result=traffic&dateGranularity=WEEKS`).reply(200, analyticsResponse)
```

Comment:
Can you use interpolation here so if we change the variables the test stays updated? `/api/analytics/site/${site_name}?from=${from}&to=${to}&dimension=${dimension}&result=${result}&dateGranularity=${date_granularity}`